repos/spack_repo/builtin/packages/python/package.py

```diff
@@ -306,6 +306,14 @@ def determine_variants(cls, exes, version_str):
         else:
             variants += "~pythoncmd"
 
+        if Version(version_str) >= Version("3.13"):
+            for exe in exes:
+                if exe.endswith("t"):
+                    variants += "+freethreading"
+                    break
+            else:
+                variants += "~freethreading"
```

Comment:
This is not exactly the recommended approach: https://docs.python.org/3/howto/free-threading-python.html#identifying-free-threaded-python

But maybe it works and is simple? Could also call sysconfig if needed.

Reply:
I can't quite see what `exes` contains, but I guess the assumption is that there's only one `python` package installed? There should be a `python3.14t` or some such name for free-threading indeed. In that case: seems reasonable.

If you'd use `sys._is_gil_enabled()` that would have to be a `subprocess` call I guess, since you want it for the target interpreter and not the one executing this `package.py`?